test/e2e/clusteradm/README.md

```diff
@@ -0,0 +1,71 @@
+[comment]: # ( Copyright Contributors to the Open Cluster Management project )
+# How to write e2e test
+
+`TestE2eConfig` has 3 public methods: 
+
+```
+RESTConfig() 
+Cluster()
+Clusteradm() 
+```
+
+---
+## RESTConfig() 
+this method return RESTConfig of hub.
+
+---
+## Cluster()
+You can get cluster informations by call Cluster(). Such as:
+```
+// name of hub
+e2e.Cluster().Hub().Name()
+
+// context of managedcluster1
+e2e.Cluster().ManagedCluster1().Context()
```

Comment:
Is this based on the assumption that all tests are run on kind? Is it possible that we have another kuberentes (non-kind) to run e2e. It should be also doc'ed here.

Reply:
yes, assuming that we use kind to start clusters, i will add comments to explain it.